third_party/move/move-compiler-v2/src/options.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
```

Comment:
See commenn on previous file.  This should probably say:

// Copyright (c) Aptos Labs, Inc.
// Copyright (c) The Diem Core Contributors
// Copyright (c) The Move Contributors

No need to reference "Parts of the project" or Meta, as far as I can tell, since this is derived from move-to-yul/src/options.rs.